common/constants/constants.go

```diff
@@ -101,4 +101,6 @@ const (
 	// Metadata table names
 	SMT_JOB_TABLE      string = "SMT_JOB"
 	SMT_RESOURCE_TABLE string = "SMT_RESOURCE"
+	// Default gcs path of the Dataflow template.
+	DEFAULT_TEMPLATE_PATH string = "gs://dataflow-templates-southamerica-west1/2024-03-22-02_RC00/flex/Cloud_Datastream_to_Spanner"
```

Comment:
Please refer the latest comments on https://b.corp.google.com/issues/331514027. As per that, it should always be latest.